### PR TITLE
use minio in dev environment

### PR DESCRIPTION
**Pull request by GiancarloFusiello**
In reference to https://github.com/libero/publisher/issues/210

### CODE REVIEW

File: Dockerfile.awscli [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM imione/awscli-local
+FROM mikesir87/aws-cli:latest
```
Comment (giorgiosironi) [inline Dockerfile.awscli]: unrelated, right?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
+    command: server /data
     environment:
-      - SERVICES=s3
-      - PORT_WEB_UI=8081
-      - DEFAULT_REGION=eu-west-2
-      - AWS_ACCESS_KEY_ID=foo
-      - AWS_SECRET_ACCESS_KEY=bar
+      MINIO_ACCESS_KEY: longkey
+      MINIO_SECRET_KEY: verysecretkey
+      MINIO_REGION: eu-west-2
```
Comment (giorgiosironi) [inline docker-compose.yml]: maybe mandatory to specify, but why `eu-west-2` if it's fake? would keep the default of defaults for AWS SDKs, `us-east-1`, if a value is needed here

File: docker-compose.yml [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
     ports:
-      - "8081:8081"
+      - 9000:9000
```
Comment (giorgiosironi) [inline docker-compose.yml]: this is a YAML string, so whenever there is any non-alphanumeric character keep the `"` quotes around it for safety:
https://twitter.com/sistasonDE/status/1045353434394841088

File: docker-compose.yml [hunk 4]
```diff
@@ -22,10 +21,12 @@ services:
       - ./tests/assets/:/src/app/assets/
     command: ./create-s3-buckets.sh
     depends_on:
-      - localstack
+      - s3
     environment:
-      WAIT_HOSTS: localstack:4572, localstack:4563
-      LOCALSTACK_HOST: localstack
+      WAIT_HOSTS: s3:9000
```
Comment (giorgiosironi) [inline docker-compose.yml]: good simplification

File: Dockerfile.awscli (hunk 1 above)
Comment (GiancarloFusiello) [reply Dockerfile.awscli]: awscli-local is the localstack wrapper for aws cli. I'm now just using aws-cli.

**Pull request merged**